utils/owner.go

```diff
@@ -0,0 +1,16 @@
+package utils
+
+import "os"
+
+// GetOwner determines the owner name from a provided parameter or environment variables.
+// It first checks the provided owner parameter, then falls back to the SUDO_USER
+// environment variable, and finally the USER environment variable.
+func GetOwner(owner string) string {
+	if owner != "" {
+		return owner
+	}
```

Comment:
what is the logic behind providing the `owner` as an input to the function if the function returns it immediately and does not use it for anything else?

Reply:
[25fa339](https://github.com/srl-labs/containerlab/pull/2583/commits/25fa3391d1e10cd9c208839061e8361de66a2d1e) just a matter of taste, so no biggue